packages/supabase_flutter/README.md

```diff
@@ -252,86 +326,238 @@ myChannel.on(
 ### [Storage](https://supabase.com/docs/guides/storage)
 
 ```dart
-import 'package:supabase_flutter/supabase_flutter.dart';
-
-final supabase = Supabase.instance.client;
-
-class MyWidget extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return ElevatedButton(
-      onPressed: () {
-        final file = File('example.txt');
-        file.writeAsStringSync('File content');
-        supabase.storage
-            .from('my_bucket')
-            .upload('my/path/to/files/example.txt', file);
-      },
-      child: const Text('Upload'),
-    );
-  }
-}
+final file = File('example.txt');
+file.writeAsStringSync('File content');
+await supabase.storage
+  .from('my_bucket')
+  .upload('my/path/to/files/example.txt', file);
+
+// Use the `uploadBinary` method to upload files on Flutter web
+await supabase.storage
+  .from('my_bucket')
+  .uploadBinary('my/path/to/files/example.txt', file.readAsBytesSync());
 ```
 
 ### [Edge Functions](https://supabase.com/docs/guides/functions)
 
 ```dart
-import 'package:supabase_flutter/supabase_flutter.dart';
+final data = await supabase.functions.invoke('get_countries');
+```
 
-final client = Supabase.instance.client;
+## Deep links
 
-class MyWidget extends StatefulWidget {
-  const MyWidget({Key? key}) : super(key: key);
+### Why do you need to setup deep links
 
-  @override
-  State<MyWidget> createState() => _MyWidgetState();
-}
+You need to setup deep links if you want your native app to open when a user clicks on a link. User clicking on a link and the app opens up happens in a few scenarios when you use Supabase auth, and in order to support those scenarios, you need to setup deep links.
 
-class _MyWidgetState extends State<MyWidget> {
-  // Persisting the future as local variable to prevent refetching upon rebuilds.
-  final _future = client.functions.invoke('get_countries');
+### When do you need to setup deep links
 
-  @override
-  Widget build(BuildContext context) {
-    return FutureBuilder(
-      future: _future,
-      builder: (context, snapshot) {
-        // return your widget with the data from snapshot
-      },
-    );
-  }
-}
+You need to setup deep links when want any of the followings
```

Comment:
```suggestion
```
Can't this be removed, because it's the same as the heading?

Reply:
Sure, sounds good!